includes/functions/functions_search.php

```diff
@@ -162,6 +162,8 @@ function zen_build_keyword_where_clause($fields, $string, $startWithWhere = fals
         global $db, $zco_notifier;
 
         $zco_notifier->notify('NOTIFY_BUILD_KEYWORD_SEARCH', '', $fields, $string);
+        if (empty($string)) { return ''; }
```

Comment:
please use the PSR-12 coding style.

Reply:
No problem, I presume you mean to break it over three lines.

Reply:
coding style has been a hotly debated topic in the past.  what i mean is here:

https://www.php-fig.org/psr/psr-12/

yes, it should be over 3 lines; but i would guess that your editor may be able to help you with this.